apps/pod-issuer-client/README.md

```diff
@@ -0,0 +1,28 @@
+## Getting Started
```

Comment:
Same as elsewhere, I wonder if this file can be deleted, or reduced to a pointer to the top-level README.